SignalServiceKit/src/Loki/Utilities/FriendRequestExpireJob.swift

```diff
@@ -150,12 +150,14 @@ public class FriendRequestExpireJob: NSObject {
                     guard message.thread.friendRequestStatus == .requestSent else {
                         // Set message to not expire, so our other logic works correctly
                         message.saveFriendRequestExpires(at: 0, with: transaction)
+                        message.saveIsFriendRequestExpired(true, with: transaction)
```

Comment:
Uh, we don't want this, because it will set `isExpired` to `true` for all other statuses